### PR TITLE
Catching any tag/push error when deploying Docker images

### PR DESCRIPTION
**Pull request by jacksondelametter**
If there was ever an error in deploying a Docker image in a Helm Chart, then an exception would be thrown and no other image would be deployed. This gives the ability to catch the exception and continue deploying Docker images.

### CODE REVIEW

File: src/main/java/org/technologybrewery/helmdependency/UploadDependenciesMojo.java [hunk 1]
```diff
@@ -4,6 +4,7 @@
 import java.nio.file.Path;
 import java.nio.file.Paths;
 import java.util.Arrays;
+import java.util.HashSet;
 import java.util.List;
```
Comment (carter-cundiff) [inline src/main/java/org/technologybrewery/helmdependency/UploadDependenciesMojo.java]: Do you still need the List import?

**Review by carter-cundiff — APPROVED** [review]

File: src/main/java/org/technologybrewery/helmdependency/UploadDependenciesMojo.java [hunk 2]
```diff
@@ -67,14 +68,19 @@ public void execute() throws MojoExecutionException {
...
         for(String dockerImage : dockerImages) {
             String newTag = String.format("%s%s", dockerUrl, dockerImage);
-            executor.executeAndLogOutput(Arrays.asList("tag", dockerImage, newTag));
-            executor.executeAndLogOutput(Arrays.asList("push", newTag));
+            try {
+                executor.executeAndLogOutput(Arrays.asList("tag", dockerImage, newTag));
+                executor.executeAndLogOutput(Arrays.asList("push", newTag));
+            }
+            catch (Exception e) {
+                getLog().error(e.getMessage());
```
Comment (carter-cundiff) [inline src/main/java/org/technologybrewery/helmdependency/UploadDependenciesMojo.java:81]: Does this not still throw an exception and stop the execution? Would think `.error()` doesn't allow you to keep pushing images?

**Review by aaron-gary — APPROVED** [review]

File: src/main/java/org/technologybrewery/helmdependency/UploadDependenciesMojo.java (hunk 2 above)
Comment (jacksondelametter) [reply src/main/java/org/technologybrewery/helmdependency/UploadDependenciesMojo.java:81]: Its just logging the message as an error, it keeps on pushing.

File: src/main/java/org/technologybrewery/helmdependency/UploadDependenciesMojo.java (hunk 1 above)
Comment (jacksondelametter) [reply src/main/java/org/technologybrewery/helmdependency/UploadDependenciesMojo.java]: Good catch we dont!

Comment (jacksondelametter) [reply src/main/java/org/technologybrewery/helmdependency/UploadDependenciesMojo.java]: Good catch we dont!

**Pull request merged**